bindings/python/docs/source/schemas.rst

```diff
@@ -51,6 +51,12 @@ For Pandas and NumPy you can do the same exact thing:
    0  string   {'start': 0}
    1  string  {'start': 10}
 
+To get a flat Pandas DataFrame with, e.g., "prop.name" and "prop.start", you can flatten the arrow_table before conversion
+
+.. code-block:: pycon
+
+   >>> df = arrow_table.flatten().to_pandas(split_blocks=True, self_destruct=True)
+   ... del arrow_table
```

Comment:
Neat! This sounds like it deserves it's own section header, like:
```
Flatten Nested Data with pyarrow.Table.flatten()
------------------------------------------------

One can flatten an arrow table using :class:`pyarrow.Table.flatten`, for example with "prop.name" and "prop.start":

.. code-block:: pycon

   >>> arrow_table = find_arrow_all(coll, {})
   >>> arrow_table = arrow_table.flatten()
   pyarrow.Table
   start: string
   prop.name: string
   prop.start: int32
   ----
   start: [["string","string"]]
   prop.name: [["foo","bar"]]
   prop.start: [[0,10]]


To get a flat Pandas DataFrame, call :class:`pyarrow.Table.flatten` before conversion:

.. code-block:: pycon

   >>> df = arrow_table.flatten().to_pandas(split_blocks=True, self_destruct=True)
   ... del arrow_table
   ... print(df)
       start  prop.name  prop.start
   0  string        foo           0 
   1  string        bar          10

   
Flatten Nested Data with Projections
------------------------------------

One can also use projections to flatten...
```

Reply:
Also it's worth pointing out that flatten() only flattens one level per call as pointed out in https://issues.apache.org/jira/browse/ARROW-4090 / https://github.com/apache/arrow/issues/20683.